Navigation bar

Wikitelmex opened the pull request:
- add navbar
- add links to navbar
- spa like page
- add contact page

wikitelmex1 committed:
Navigation bar

- add navbar
- add links to navbar
- spa like page
- add contact page

wikitelmex1 committed:
add Luxon dateTime

- add datetime down below navbar
- instaled luxon
- add luxon call
- fix navbar behavior to close collapsible menu when option selected

rloterh commented on index.html:
```diff
@@ -13,19 +13,39 @@
         <link rel="stylesheet" href="style.css">
     </head>
+
     <body>
```
- [x] The _footer_ is missing in your pages. Kindly ensure to add the footer to the pages.

**Your design:**
![image](https://user-images.githubusercontent.com/12745474/133691653-0b90474f-41e3-45d7-9dd5-eb86a4ed7a10.png)


**The wireframe design:**
![image](https://user-images.githubusercontent.com/12745474/130169854-e3b0b337-4389-40f0-b387-cd1587363afe.png)

rloterh commented on index.html:
```diff
@@ -13,19 +13,39 @@
+                    <span class="navbar-toggler-icon"></span>
+                </button>
+                <div class="collapse navbar-collapse mx-auto" id="navbarNavAltMarkup">
+                    <div class="navbar-nav ml-auto">
+                        <a onclick="Show('list')" data-bs-toggle="collapse" data-bs-target="#navbarNavAltMarkup" id="listLink" class="nav-link active" aria-current="page" href="#">List</a>
+                        <a onclick="Show('add')" data-bs-toggle="collapse" data-bs-target="#navbarNavAltMarkup" id="addLink" class="nav-link" href="#">Add new</a>
+                        <a onclick="Show('contact')" data-bs-toggle="collapse" data-bs-target="#navbarNavAltMarkup" id="contactLink" class="nav-link" href="#">Contact</a>
+                    </div>
```
- [x] The right menu elements in your navbar are missing the dividers between each menu element, as can be seen in the provided [wireframe design](https://github.com/microverseinc/curriculum-javascript/blob/main/books/m4_full_website.md). Kindly ensure to improve upon this aspect in your design, as we really want our designs to as much as possible mimic the provided wireframe template.

**Your design menu:**
![image](https://user-images.githubusercontent.com/12745474/133692251-696a1acf-1512-4537-912f-b1eeff82dbd8.png)


**The wireframe menu design:**
![image](https://user-images.githubusercontent.com/12745474/129183812-a30acaa9-4f6c-4d4e-973b-5dd83a591c8d.png)


- [ ] OPTIONAL
It will be a nice idea to structure your menu items using the `ul` and `li` tags

rloterh commented on index.html:
```diff
@@ -13,19 +13,39 @@
+        <div id="addPage" class="d-none">
             <h2>Add a book</h2>
             
             <form name="saveBook" id="saveBook">
```
- [x] Your contact information is **not centered** on the page. Kindly ensure to center the contact information, as can be seen in the screenshot o the original below:

**Your contact page:**
![image](https://user-images.githubusercontent.com/12745474/133693128-2b0b3932-b387-406e-86ac-167895b1cb5c.png)


**The original contact page:**
![image](https://user-images.githubusercontent.com/12745474/133693085-f60948d7-3daa-4d84-9873-df01559d0d4f.png)

rloterh commented on index.html:
```diff
@@ -13,19 +13,39 @@
-        <table class="table border" id="booksTable">
-           
-        </table>
+        <!--Datetime-->
```
- [x] OPTIONAL
There are comments found in your code on the repository. Kindly endeavor to remove **all** comments from your code before pushing it to the repository so as to keep your code cleaner.

Wikitelmex replied:
changed this comments to regions, to keep the code simple to read

rloterh commented on script/script.js:
```diff
@@ -3,6 +3,8 @@ const booksAr = new BooksArray();
+const myDate = document.querySelector('#myDate');
+const dt = luxon.DateTime.now().toLocaleString(luxon.DateTime.DATETIME_MED);
 
 function RemoveBooks(idIndex) {
```
- [ ] OPTIONAL
It is a nice idea to prefer _arrow functions_ over normal functions, so as to keep your _function_ syntax simpler and cleaner.

rloterh commented on script/script.js:
```diff
@@ -15,9 +17,42 @@ function AddBookM() {
   domTb.AddRowBooks(s.title, s.author, s.index);
```
- [x] OPTIONAL
The book _title_ and book _author_ input are both accepting numbers as inputs. Kindly endeavor to provide validation for the same

rloterh commented on index.html:
```diff
@@ -13,19 +13,39 @@
         <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css" integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
         <link rel="stylesheet" href="style.css">
     </head>
+
```
- [x] OPTIONAL
Kindly add a project title to your ReadMe so as to make it look professional

rloterh reviewed (changes requested):
Hi Alex & Peter, 

You did extremely great on this project! 👏 👏 
There are however a few issues that you still need to work on to go to the next project but you are almost there! 🚦 🚦 🚦  

## Good Points 👍  
- No linter errors


## Required Changes ♻️
_Kindly check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

_Kindly check the comments marked **[OPTIONAL]** under the review._


<br>
Cheers and Happy coding!👏 💻 🍷

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

wikitelmex1 committed:
Improvements required

- add footer with social networks
- add dividers to navbar
- contact module centered
- comments changed with regions to allow developers keep cleanes code when developing
- pattern to inputs in form to accept letters and numbers, but not only numbers
- add readme title

wikitelmex1 committed:
linters and pattern

wikitelmex1 committed:
styling corrections

wikitelmex1 committed:
Update style.css
